ynh-dev

```diff
@@ -99,7 +102,7 @@ elif [ "$1" = "create-env" ]; then
 elif [ "$1" = "run" ]; then
 
     DOMAIN=$2
-    VERSION='stable'
+    VERSION='stable64'
```

Comment:
I'm wondering if it's a good idea to rename these with "64" at the end. It kind of break retro-compatibility (or at least I think people are use to simply type "stable"/"testing"/"unstable"). I think simply having stable32/testing32/unstable32 for the particular 32-bit version would be good enough, no need to change the existing stable/testing/unstable.